### DISCUSSION TITLE
Facing Issue while running workflow file

### DISCUSSION THREAD

**Discussion by SutharShantanu** [athul/waka-readme#147] [Q&A] [answered]
Error occurred:- Can't find 'action.yml', 'action.yaml' or 'Dockerfile' for action 'SutharShantanu/SutharShantanu@main'.

`name: Waka Readme

on:
  # for manual workflow trigger
  workflow_dispatch:
  schedule:
    # runs at 12 AM UTC (5:30 AM IST)
    - cron: "0 0 * * *"

jobs:
  update-readme:
    name: WakaReadme DevMetrics
    runs-on: ubuntu-latest
    steps:
      - uses: SutharShantanu/SutharShantanu@main
        with:
          GH_TOKEN: ${{ secrets.GH_TOKEN }} # optional if on profile readme
          WAKATIME_API_KEY: ${{ secrets.WAKATIME_API_KEY }} # required
          ### meta
          API_BASE_URL: https://wakatime.com/api # optional
          REPOSITORY: SutharShantanu/SutharShantanu # optional
          ### content
          SHOW_TITLE: true # optional
          SECTION_NAME: waka # optional
          BLOCKS: -> # optional          
          TIME_RANGE: all_time # optional
          LANG_COUNT: 10 # optional
          SHOW_TIME: true # optional
          SHOW_TOTAL: true # optional         
          STOP_AT_OTHER: true # optional
          ### commit
          COMMIT_MESSAGE: Updated waka-readme graph with new metrics # optional
          TARGET_BRANCH: main # optional
          TARGET_PATH: README.md # optional
          COMMITTER_NAME: GitHubActionBot # optional
          COMMITTER_EMAIL: action-bot@github.com # optional
          AUTHOR_NAME: Shantanu # optional`

**Comment by nandgator** [comment, marked as answer]
Hi @SutharShantanu, thanks for trying out `athul/waka-readme` This is very similar to https://github.com/athul/waka-readme/issues/142

In [line 19 of your workflow file](https://github.com/SutharShantanu/SutharShantanu/blob/18ded66306611b58f814a5446bebcb38b98cd9bf/.github/workflows/waka-readme.yml#L19), you must have - `uses: athul/waka-readme@master`, instead of - `uses: SutharShantanu/SutharShantanu@main`

```diff
steps:
-    - uses: SutharShantanu/SutharShantanu@main
+    - uses: athul/waka-readme@master
       with:
```

**Reply by SutharShantanu** [reply]
Thanks for the help. It worked 😌
